Podcast/SearchITunesTableViewController.swift

```diff
@@ -30,10 +30,11 @@ class SearchITunesTableViewController: ViewController, UITableViewDelegate, UITa
         mainScrollView = tableView
         
         loadingIndicatorView = createLoadingAnimationView()
-        loadingIndicatorView!.center = CGPoint(x: UIScreen.main.bounds.width / 2, y: UIScreen.main.bounds.height / 2)
         view.addSubview(loadingIndicatorView!)
+        loadingIndicatorView?.snp.makeConstraints { make in
```

Comment:
why is this loadingIndicator an optional? EmptyStateTableView now has a loading indicator within it so we don't need this loading indicator.